ref/test/CMakeLists.txt

```diff
@@ -72,68 +53,38 @@ endforeach(FILENAME)
-
-# 0.0.3 input
-add_test(NAME regression_0.0.3
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.3.nl 2>&1 | tee test_output/yppm_0.0.3.log.out")
-add_test(NAME compare_0.0.3
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.3.log.out test_output/yppm_0.0.3.test)
-
-# 0.0.6 input
-add_test(NAME regression_0.0.6
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.6.nl 2>&1 | tee test_output/yppm_0.0.6.log.out")
-add_test(NAME compare_0.0.6
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.6.log.out test_output/yppm_0.0.6.test)
-
-# 0.0.8 input
-add_test(NAME regression_0.0.8
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.8.nl 2>&1 | tee test_output/yppm_0.0.8.log.out")
-add_test(NAME compare_0.0.8
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.8.log.out test_output/yppm_0.0.8.test)
-
-# 0.1.3 input
-add_test(NAME regression_0.1.3
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.1.3.nl 2>&1 | tee test_output/yppm_0.1.3.log.out")
-add_test(NAME compare_0.1.3
-         COMMAND ${COMPARE_SH} test_output/yppm_0.1.3.log.out test_output/yppm_0.1.3.test)
-
-# 0.1.6 input
-add_test(NAME regression_0.1.6
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.1.6.nl 2>&1 | tee test_output/yppm_0.1.6.log.out")
-add_test(NAME compare_0.1.6
-         COMMAND ${COMPARE_SH} test_output/yppm_0.1.6.log.out test_output/yppm_0.1.6.test)
-
-# 0.1.7 input
-add_test(NAME regression_0.1.7
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.1.7.nl 2>&1 | tee test_output/yppm_0.1.7.log.out")
-add_test(NAME compare_0.1.7
-         COMMAND ${COMPARE_SH} test_output/yppm_0.1.7.log.out test_output/yppm_0.1.7.test)
-
-# 0.1.15 input
-add_test(NAME regression_0.1.15
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.1.15.nl 2>&1 | tee test_output/yppm_0.1.15.log.out")
-add_test(NAME compare_0.1.15
-         COMMAND ${COMPARE_SH} test_output/yppm_0.1.15.log.out test_output/yppm_0.1.15.test)
-
-# 0.0.7 input
-add_test(NAME regression_0.0.7
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.7.nl 2>&1 | tee test_output/yppm_0.0.7.log.out")
-add_test(NAME compare_0.0.7
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.7.log.out test_output/yppm_0.0.7.test)
-
-# 0.0.9 input
-add_test(NAME regression_0.0.9
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.9.nl 2>&1 | tee test_output/yppm_0.0.9.log.out")
-add_test(NAME compare_0.0.9
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.9.log.out test_output/yppm_0.0.9.test)
-
-# 0.0.12 input
-add_test(NAME regression_0.0.12
-         COMMAND bash -c "set -o pipefail; ../src/yppm test_input/yppm_0.0.12.nl 2>&1 | tee test_output/yppm_0.0.12.log.out")
-add_test(NAME compare_0.0.12
-         COMMAND ${COMPARE_SH} test_output/yppm_0.0.12.log.out test_output/yppm_0.0.12.test)
+# 0.1.7 input_10
```

Comment:
Please add the original tests back in for the two cases you decided to keep.

Reply:
Done.